hedyorg/hedy
# 🪲 Debug play

Adds the `play` builtin function to both `run` and  `debug` paths, so now we're able to debug the play command. Also fixes the highlighting of variables within the play command, as well as fixing the highlighting of the lines containing the play command while debugging.

Fixes  #5111 

**How to test**
*  Automated tests should pass
* Go to level 2 and debug this code:

```
note is C1
play note
play C2
```

Also check that the `note` variable is colored blue.
